src/modules/users/users.controller.ts

```diff
@@ -86,6 +86,19 @@ export class UsersController {
     return me;
   }
 
+  @Patch('me')
+  async updateMe(
+    @Body()
+    payload: Pick<UpdateUserDto, 'fullName' | 'gitHubUrl' | 'linkedInUrl'>,
+    @GetUser() user?: CurrentUser,
+  ): Promise<User> {
+    if (!user?.id) {
```

Comment:
@grizlizli, I guess we don't need to check the user because the auth middleware handles it, right?